### PR TITLE
Improve facilities results

### PR DESCRIPTION
**Pull request by gkop**

### CODE REVIEW

File: app/assets/javascripts/components/map.js [hunk 1]
```diff
@@ -9,9 +9,10 @@ $(function () {
       zoom: 6,
     });
 
+    var limit = 12;
     $('.map-finder').on('submit', function (e) {
       e.preventDefault();
-      $.get('/api/facilities.json?' + $(e.target).serialize())
+      $.get('/api/facilities.json?' + $(e.target).serialize() + "&limit=" + limit)
```
Comment (KaoruDev) [inline app/assets/javascripts/components/map.js]: does the endpoint only respond with the amount set by the limit? if so we can remove the `first` method call below and do:

```
}).each(function (facility) {...
```

Comment (KaoruDev) [inline app/assets/javascripts/components/map.js]: if this is also true we can set the `limit` as a `hidden` input tag on the form and `$(..).serialize()` will take care of transforming it into a url query.

```
<input type="hidden" name="limit" value="12">
```

Comment (KaoruDev) [inline app/assets/javascripts/components/map.js]: oh yep and it does haha

File: lib/tasks/geocode_facilities.rake [hunk 2]
```diff
@@ -27,15 +27,15 @@ task :geocode_facilities => :environment do
     if first
       # trust google ;)
       # in all cases that I checked, Google's was a legit match
-      update_facility(facility, first, query)
+      update_facility(facility, first, query, false)
```
Comment (KaoruDev) [inline lib/tasks/geocode_facilities.rake]: why don't we _always_ use `Geocoder` if it gets us more accurate data?

Comment (gkop) [inline lib/tasks/geocode_facilities.rake]: Hehe. False here means it did not come through for us :( I am _trying_ everything, but Google's API really falls short with the business names. An API that correlated business names and counties would be awesome, but I didn't find one readily.

**Pull request merged**